### ISSUE TITLE
Tenor gifs saved as png

### ISSUE THREAD

**Issue by Tokariew** [Tyrrrz/DiscordChatExporter#817]
### Version

latest docker

### Flavor

CLI (Command Line Interface)

### Export format

TXT

### Details

running command below to make archive of my chats… expected that animated gifs will be saved as gif, but some of them are saved as png…

The optional `:Z` option is for selinux when using with docker/podman

### Steps to reproduce

- `podman run --rm --name discord -v /home/tokariew/Documents/discord:/app/out:Z docker.io/tyrrrz/discordchatexporter:latest exportdm -t "skipped" -f PlainText --media --reuse-media`

**Comment by Tyrrrz** [comment]
Related: #683

**Comment by Tyrrrz** [comment]
Wait, are gif embeds even saved in plain text export?

**Comment by Tokariew** [comment]
Why they shouldn't be, with the command above?
I get something like that in my exported txt file

```
[2021-07-26 18:35] User#number
https://tenor.com/view/tv-shows-television-tackle-hug-hug-glomping-gif-14859564

{Embed}
https://tenor.com/view/tv-shows-television-tackle-hug-hug-glomping-gif-14859564
User.txt_Files/tenor-E0914.png
```

with the following command:

```
podman run --rm --name discord -e TZ=Europe/Warsaw -v /home/tokariew/Documents/discord:/app/out:Z docker.io/tyrrrz/discordchatexporter:latest exportdm -t "KEY" -f PlainText --dateformat "yyyy-MM-dd HH:mm" --media --reuse-media -o "%a/%C.txt"
```

**Comment by Tyrrrz** [comment]
Ah yes, it resolves as an embed. Makes sense.

**Comment by MrZeusTheCoder** [comment]
Is there any update on this? I would be willing to help, but I don't have any C# in my background. :sweat_smile: 

I looked around the code a bit. Would all it require is adding a new embed type? I might take a stab at it if so.

**Comment by Tyrrrz** [comment]
The issue is that Tenor "gifs" are actually webm videos and they are rendered on Discord using a special embed. It might not be that easy to get a link to the underlying video.

**Comment by sus-impost0r** [comment]
Would probably need some screenscraping or API calls. IMO, this is outside the scope of DCE.

**Comment by MrZeusTheCoder** [comment]
Is this closed by #919 and #927?

**Comment by Tokariew** [comment]
yes it seems to be fixed on newest version. The gif are being saved correctly ;)
